src/main/java/org/ow2/proactive/catalog/service/CatalogObjectGrantService.java

```diff
@@ -418,7 +418,7 @@ public void deleteAllCatalogObjectGrantsByBucketIdAndObjectId(long bucketId, lon
     public List<CatalogObjectGrantMetadata> deleteAllCatalogObjectGrantsAssignedToABucket(String bucketName,
             String catalogObjectName) {
         long bucketId = bucketRepository.findOneByBucketName(bucketName).getId();
-        long catalogObjectId = getCatalogObjectId(bucketName, catalogObjectName);
+        long catalogObjectId = getCatalogObject(bucketName, catalogObjectName).getId();
```

Comment:
why getCatalogObject returns here a long and below a CatalogObjectRevisionEntity ?